inc/endpoints/class-components-registry-endpoint.php

```diff
@@ -24,7 +25,7 @@ public function register_rest_routes() {
 			[
 				'methods'  => \WP_REST_Server::READABLE,
 				'callback' => function() {
-					return get_registry()->get_registered_components();
+					return Components\get_registry()->get_registered_components();
```

Comment:
👍 Do we need a general helper function for this?

Reply:
I don't think so, and if so, we can tackle it later - this is just fixing the namespace change after the refactor.